src/herder/TxQueueLimiter.cpp

```diff
@@ -90,7 +90,26 @@ TxQueueLimiter::removeTransaction(TransactionFrameBasePtr const& tx)
     mQueueSizeOps -= txOps;
 }
 
-bool
+// compute the fee bid that `tx` should have in order to beat
+// a transaction `ref` with fee bid `refFeeBid` and `refNbOps` operations
+int64
+computeBetterFee(TransactionFrameBasePtr const& tx, int64 refFeeBid,
+                 uint32 refNbOps)
+{
+    constexpr auto m = std::numeric_limits<int64>::max();
+
+    int64 minFee = m;
+    int64 v;
+    if (bigDivide(v, refFeeBid, tx->getNumOperations(), refNbOps,
+                  Rounding::ROUND_UP) &&
```

Comment:
This is a pretty minor detail, but I think this should be `Rounding::ROUND_DOWN`. Consider the following:
```
refFeeBid = 100
tx->getNumOperations() = 2
refNbOps = 3

-> v = ceil(66.66) -> minFee = 68
```
but from fee rate compare
```
201 = 67 * 3 > 100 * 2 = 200
```
so `minFee` should have been 67.

The `v + 1` is still right though, which can be seen from considering the case `100, 1, 1`.

Reply:
indeed